src/app/grants/data/submissionCriteriaData.tsx

```diff
@@ -0,0 +1,20 @@
+import { Code, BracketsAngle, Users } from '@phosphor-icons/react/dist/ssr'
+
+export const submissionCriteriaData = [
+  {
+    title: 'Open-source',
+    description: 'All projects must be open-sourced.',
+    Icon: Code,
```

Comment:
Everywhere else also:

```suggestion
    icon: Code,
```